packages/protocol/contracts/L2/TaikoL2.sol

```diff
@@ -159,8 +159,7 @@ contract TaikoL2 is CrossChainOwned, TaikoL2Signer, ICrossChainSync {
         if (token == address(0)) {
             to.sendEther(address(this).balance);
         } else {
-            IERC20 t = IERC20(token);
-            t.transfer(to, t.balanceOf(address(this)));
+            IERC20(token).transfer(to, IERC20(token).balanceOf(address(this)));
```

Comment:
Should not we use SafeERC20 ? As like in AssignmentHook ?
https://github.com/taikoxyz/taiko-mono/pull/15423/files